mne/commands/mne_watershed_bem.py

```diff
@@ -37,9 +37,6 @@ def run():
                       action="store_true")
     parser.add_option("-p", "--preflood", dest="preflood",
                       help="Change the preflood height", default=None)
-    parser.add_option("--copy", dest="copy",
-                      help="Use copies instead of symlinks for surfaces",
-                      action="store_true")
```

Comment:
@larsoner all this bem related things that were added to the PR. Were they needed?? I mean, they are unrelated to the PR. But maybe you wanted them.

Reply:
They were needed when I created the 3-layer BEM for fsaverage, so yes please keep